tests/ha/sle11_cluster_init.pm

```diff
@@ -3,12 +3,34 @@ use testapi;
 use autotest;
 
 sub run() {
-    send_key 'shift-ctrl-alt-g';
-    type_string "sleha-init -y -s /dev/disk/by-id/scsi-1LIO-ORG.FILEIO:348cfd84-58a1-426e-9797-e22f04cf207f\n";
-    assert_screen 'cluster-init',60;
+    if (check_var("WORKER_CLASS", "ha_master")) {
+        type_string "sleha-init -y -s /dev/disk/by-id/scsi-1LIO-ORG.FILEIO:348cfd84-58a1-426e-9797-e22f04cf207f\n";
+        assert_screen 'cluster-init',60;
+        type_string "crm status\n";
+        assert_screen 'cluster-status';
+        send_key 'ctrl-l';
+        mutex_create('cluster-init');
```

Comment:
Does it work for you as is? mutex_create is exported by lockapi.pm which I don't see being imported. I'm missing 'use lockapi'.